<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>单词自动补全
&quot;alpha&quot; &quot;omega&quot; &quot;omicron&quot; &quot;zeta&quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fileLineEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="dirLineEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>250</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>文件自动补全</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>220</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>目录自动补全</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
